crypto/bio/bss_conn.c

```diff
@@ -425,6 +435,8 @@ static long conn_ctrl(BIO *b, int cmd, long num, void *ptr)
                     ret = -1;
                     break;
                 }
+            } else if (num == 4) {
```

Comment:
magic numbers :-(

Reply:
This follows the rest of the OpenSSL code in this function. It's a bit out of scope to get rid of magic numbers in this PR (IMO).